http/http.go

```diff
@@ -76,15 +140,16 @@ func PostHelper(ctx context.Context, httpClient *http.Client, stats *statsd.Clie
 		req.Header.Set("Content-Encoding", "deflate")
 	}
 	// we only make http requests at flush time, so keepalive is not a big win
-	req.Close = true
+	// req.Close = true
```

Comment:
not sure if intentionally commented out, rather than deleted

Reply:
Oops, had that locally and hadn't pushed. Ty!